README.md

```diff
@@ -1,81 +1,101 @@
 # CakePHP Gearman plugin
-A gearman plugin for CakePHP 3.x
 
-## How to use
+Gearman task offloading for CakePHP 3.
 
-### Installation
+## Requirements
+
+- CakePHP 3.0+
+- [Gearman](http://gearman.org)
+
+## Why use this plugin?
+
+Use this plugin to drastically reduce page load times by offloading
+time consuming processes (like sending emails and resizing uploaded images) to
+the Gearman Job Server on your local system.
```

Comment:
Yeah, eff9b4930470f718e4805ee60cca3f58234017f9. Thanks for the improvements! :+1: